### DISCUSSION TITLE
Function key and keys backlighting

### DISCUSSION THREAD

**Discussion by Diego-JRH** [seerge/g-helper#1843] [Ideas]
In some laptops like the MSI I had or the new lenovo legion laptops for example, when you press the Fn key, all the keys that have secondary functions get backlit while all the others turns off while is being pressed and everything turns back when it's released. I think it is a nice addition and I missed it when I switched to Asus so I was wondering if adding that function is even possible or if it is within the possibilities of this app.

**Comment by seerge** [comment]
@Diego-JRH that's not possible, cause FN key by itself doesn't emit any keycode, so you simply don't know when it's pressed

**Reply by Diego-JRH** [reply]
I expected it, it must be related to the bios or something I guess
